src/libraries/Native/Windows/System.IO.Compression.Native/zlib-ng/LICENSE.md

```diff
@@ -0,0 +1,19 @@
+(C) 1995-2013 Jean-loup Gailly and Mark Adler
```

Comment:
Presumably we also need to update https://github.com/dotnet/runtime/blob/main/THIRD-PARTY-NOTICES.TXT ?